### ISSUE TITLE
Feature Request: Add hotkey for "Start" action (context menu command)

### ISSUE THREAD

**Issue by ErdLingMS** [Maxstupo/ydl-ui#92]
When a download task fails, one has to right click the task in the list and click on "Start" to resume it. Depending on circumstances, this has to be done multiple times for one task. Unlike most other commands, there is no keyboard hotkey for this action.

My suggestion is to add a hotkey to more easily retry/resume those failed download tasks.

**Comment by Maxstupo** [comment]
It's possible to select multiple download tasks and then right click — start.

**Comment by ErdLingMS** [comment]
Yes, but that is not what I am requesting. There is no keyboard shortcut to do it. Having to use the mouse for this is cumbersome. Regardless of whether it is just one or multiple downloads that we are talking about.

Btw, your reply gave me an idea, which especially is relevant when seletcing multiple tasks at once: Please, when you add a hotkey, do NOT have Start and Stop have the same hotkey. That could lead to severe confusion. So please, either leave no hotkey for "Stop", or a different one. So if you have selected 3 downloads and 1 of them is still running while the other two are stopped, the hotkey can start those two while doing nothing to the one that is still running.

**Comment by Maxstupo** [comment]
I like that idea. Maybe `Ctrl+S` for starting, and `Ctrl+P` for stopping. Might also be a good idea to split the start/stop button in the right click context menu.
